testsuite/tests/test.rs

```diff
@@ -2,6 +2,7 @@
 #![no_main]
 
 use {{crate_name}} as _; // memory layout + panic handler
+use defmt::{assert, assert_eq};
```

Comment:
`assert` seems unused

Reply:
it doesn't show up in the diff but there was already an `assert_true` unit test that was using `core::assert`. With this change, it's now using the defmt version

Reply:
Ah, okay